.github/workflows/release-prepare.yml

```diff
@@ -26,12 +26,12 @@ jobs:
       - uses: ./.github/workflows/rust-cache
       - uses: actions/setup-java@v3
         with:
-          java-version: '17'
+          java-version: '11'
           distribution: 'temurin'
       - name: Configure Git User
         run: |
-          git config user.email "actions@github.com"
-          git config user.name "GitHub Actions"
+          git config user.email 'actions@github.com'
+          git config user.name 'GitHub Actions'
       - name: Run mvn release
         run: mvn --batch-mode release:prepare -DreleaseVersion=${{ github.event.inputs.releaseVersion }} -DdevelopmentVersion=${{ github.event.inputs.developmentVersion }}
```

Comment:
I kind of expect this to fail? 🤔 Can you dry-run it locally?

Reply:
It appears to work - I get a local failure because I can't git push.